.travis.yml

```diff
@@ -78,4 +78,33 @@ jobs:
         - PATH="$HOME/miniconda/bin:$PATH"
       install: .travis/install_conda.sh "Miniconda3-latest-Linux-x86_64.sh"
       script:
+        - .travis/conda_build_and_deploy.sh --label 'dev' --python 3.6
+    - stage: "Release"
+      name: "release conda linux"
+      if: NOT type = cron AND branch = master AND NOT type = pull_request
+      on:
+        tags: true
+      git:
+        depth: false
+      env:
+        - PATH="$HOME/miniconda/bin:$PATH"
```

Comment:
To further remove duplication this could be moved to the global env at the top of the file (on jobs with no Minicoda it would just have no effect).